build/yamls/antrea.yml

```diff
@@ -80,7 +80,7 @@ rules:
   - update
   - delete
 - apiGroups:
-  - networkpolicy.antrea.io
+  - networkpolicy.antrea.tanzu.vmware.com
```

Comment:
@jianjuns do you still we should change the group name to "networking.antrea.tanzu.vmware.com" here?

Reply:
"networking" sounds good to me.
Do you think we should differentiate internal message in some way (considering we might have user facing API, and we already have CLI API)?
@abhiraut @salv-orlando: any opinion?

Reply:
@jianjuns @tnqn @abhiraut @salv-orlando  We have confirmed with Tim that  API group to be: antrea.tanzu.vmware.com. I am going to change it to "networking.antrea.tanzu.vmware.com" if there is no other comment.

Reply:
Have updated accordingly

Reply:
We could use networking.antrea.tanzu.vmware.com in this PR, but I still hope to discuss further the appropriate names for "internal" API (controller-agent messages), and API for CLI (are they really specific to CLI).